### PR TITLE
Introduce cost center creation showcase app

### PR DESCRIPTION
**Pull request by Emdee89**

### CODE REVIEW

**Comment by fwilhe** [comment]
I suspect the file `integration-tests/dev_jco_rfc.trc` should not have been committed, right?

**Comment by Emdee89** [comment]
Trace file integration-tests/dev_jco_rfc.trc removed

**Comment by sacnl** [comment]
Thanks, can you add this app as one line to the root README.md file where the applications of this repo are listed?

Are there any non-obvious prerequisites that the S/4 system to connect to has to fulfill?

**Comment by Emdee89** [comment]
@sacnl : 
(1) README.md adjusted
(2) None.

File: Cost-Center-Creation-Neo/application/src/main/java/com/sap/opensap/CostCenterCreationServlet.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import com.sap.cloud.sdk.s4hana.connectivity.rfc.BapiQueryResult;
+import com.sap.cloud.sdk.s4hana.connectivity.rfc.Table;
+
+@WebServlet("/cost-center-creation")
+public class CostCenterCreationServlet extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = CloudLoggerFactory.getLogger(CostCenterCreationServlet.class);
+
+    @Override
+    protected void doGet(final HttpServletRequest request, final HttpServletResponse response)
```
Comment (gavrilova-ev) [inline Cost-Center-Creation-Neo/application/src/main/java/com/sap/opensap/CostCenterCreationServlet.java]: Shouldn't that be done on doPost? Creation of cost centers on HTTP get is confusing.

**Pull request merged**